docker/docker-compose.setup.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.4'
+
+services:
+    setup:
+        image: sylius/server:latest
+        env_file:
+            - .env
+        depends_on:
+            - sylius
+        command: "php ./bin/console sylius:install"
```

Comment:
After successfully running this script, the assets are still missing on a fresh install. Probably the best would be to run the Yarn assets build command as well.